src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -1755,8 +1670,8 @@ configure_cu(struct sched_cmd *cmd, int cu_idx)
 	cmd->exectime = cu_idx_to_timeout(cmd->ddev, cmd->cu_idx);
 	cmd->check_timeout = cmd->exectime > 0 ? 1 : 0;
 
-	/* start CU at base + 0x0 */
-	iowrite32(0x1, virt_addr);
+	zocl_cu_start(cu);
```

Comment:
I have some additional question, regrading the cu_done and cu_start.

zocl_cu_start iowrite32(0x1, cu_core->vaddr); 
list_add_tail(&cmd->rq_list, &cu->running_queue); 
 
ert_configure_cu doesn't add it to the running queue? I guess it changed the current behavior of handling cu? aka, set to 0x1 wait for 0x6? now we keep setting it?

Reply:
ert_configure_cu should do the same. Let me fix it.
We didn't keep setting it. Before submit the command to the CU. Scheduler need to get a credit from CU. The old behavior will like the credit of the CU is ways 1.
But if CU support Command Queue, the credit of the CU is > 1. So, before the credit burns out, the scheduler could always get a credit for the command and submit the command to a CU.

Reply:
LGTM